app/src/main/java/com/stevesoltys/seedvault/SecretCodeReceiver.kt

```diff
@@ -13,8 +13,7 @@ private const val RESTORE_SECRET_CODE = "7378673"
 class SecretCodeReceiver : BroadcastReceiver() {
 
     override fun onReceive(context: Context, intent: Intent) {
-        val host = intent.data.host
-        if (RESTORE_SECRET_CODE != host) return
+        if (intent.data?.host != RESTORE_SECRET_CODE) return
```

Comment:
This is an unrelated change, because I saw the lint warning about `intent.data` possibly being `null`.